python/test/simple.py

```diff
@@ -24,44 +24,44 @@
 # password can be checked on the server for password rules
 # (e.g. occurence in common password lists), has the drawback that the
 # password is exposed to the server.
-rec, export_key0 = opaque.Register(pwd,cfg,ids,skS=None)
+rec, export_key = opaque.Register(pwdU,cfg,ids,skS=None)
```

Comment:
by calling it export_key0 and later having also an export_key, we can do an assert(==)

Reply:
I added assertions.